### DISCUSSION TITLE
Export vectorization tool output to DXF file format

### DISCUSSION THREAD

**Discussion by Abworwood** [arkypita/LaserGRBL#2163] [Ideas]
I have a CNC Plasma cutter and Router I'm thinking about adding a laser as well. Currently I use sheetcam for creating all of my G-code for my plasma cutter but it needs a DXF file to import into it. Your vectorization tool works awesome it outputs the image perfectly into a vector. Is there a way to add exporting the vector to a DXF file type.

**Comment by arkypita** [comment]
I suggest to use inkscape to vectorize (inkscape use same algorithm as LaserGRBL) and i think inkscape can export dxf

**Comment by Abworwood** [comment]
That is what I try and use sometimes but I always have to fix up the trace
because it doesn't look anything like it. I used the vectorize tool on
laser grbl and it worked perfectly. I know they say they use the same
potrace algorithm but for some reason laser grbl was the best that I could
find. I tried a dozen or more tracing and converter programs. I've attached
the output of laser grbl default settings the best I could get it on
Inkscape I tried every setting combination.
Thanks Anthony
